zproject/urls.py

```diff
@@ -50,6 +47,11 @@
     update_realm_custom_profile_field,
     update_user_custom_profile_data,
 )
+from zerver.views.development.dev_login import (
+    api_dev_fetch_api_key,
+    api_dev_list_users,
+    dev_direct_login,
+)
```

Comment:
Can you move these to `dev_urls.py` perhaps in a second commit? 

Otherwise looks great.